### ISSUE TITLE
Accuracy does not match as listed in paper

### ISSUE THREAD

**Issue by saniazahan** [LZU-SIAT/PCRP#1]
Hi, I ran your model for 10 epoch with everything unchanged for NTU-60 cross subject data. In your paper you mentioned your accuracy is 53.9 at epoch 5 but I only got max accuracy 45.7 in epoch 10. Could you tell me how did you get the higher accuracy.

**Comment by Mikexu007** [comment]
Thanks for your questions.&nbsp;From our experiment record&nbsp;of 53.9 accuracy on NTU 60 Cross-Subject, we set :

learning_rate = 0.001

bi_flag&nbsp;= False

reverse_flag = True

prototype = True

num_clusters = &nbsp;&nbsp;&nbsp;[90,120,150]

negative_r = 20

batch_size = 64 

random_seed = 55.

&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Sania Zahan"<notifications@github.com&gt;; 
发送时间: 2021年1月10日(星期天) 晚上11:02
收件人: "LZU-SIAT/PCRP"<PCRP@noreply.github.com&gt;; 
抄送: "Subscribed"<subscribed@noreply.github.com&gt;; 
主题: [LZU-SIAT/PCRP] Accuracy does not match as listed in paper (#1)





 
Hi, I ran your model for 10 epoch with everything unchanged for NTU-60 cross subject data. In your paper you mentioned your accuracy is 53.9 at epoch 5 but I only got max accuracy 45.7 in epoch 10. Could you tell me how did you get the higher accuracy.

**Comment by saniazahan** [comment]
Well these parameters are bit different then that in the code. I will try it. Also, could you please explain the effect of negative_r, is it arbitrary or there is reason to choose different value for different dataset. Thanks

**Comment by Mikexu007** [comment]
> Well these parameters are bit different then that in the code. I will try it. Also, could you please explain the effect of negative_r, is it arbitrary or there is reason to choose different value for different dataset. Thanks

We run couple experiments on negative_r, but we still do not know the effect since we have limited GPU sources and we can not set batch_size too large (batch_size limits negative_r).  Perhaps you can refer to the original work  "Prototypical Contrastive Learning of Unsupervised Representations" to find answers (It seems like that  work does not explore the effect of negative_r either).

**Comment by Mikexu007** [comment]
> Well these parameters are bit different then that in the code. I will try it. Also, could you please explain the effect of negative_r, is it arbitrary or there is reason to choose different value for different dataset. Thanks

For N-UCLA, we set negative_r = 20, batch_size = 16.
For NTU 60, we set negative_r = 20, batch_size = 64.
For NTU 120,  we set negative_r = 60, batch_size = 64

**Comment by dingli93** [comment]
> Well these parameters are bit different then that in the code. I will try it. Also, could you please explain the effect of negative_r, is it arbitrary or there is reason to choose different value for different dataset. Thanks

Have you reproduced the experiment results listed in the paper?

**Comment by saniazahan** [comment]
@dingli93 I haven't and I had to keep it aside for the time being. Please let me know if you can.
